Plugins/GeoModel/src/GeoModelDetectorSurfaceFactory.cpp

```diff
@@ -55,24 +59,37 @@ void Acts::GeoModelDetectorSurfaceFactory::construct(
     auto qFPV = geoModelTree.geoReader
                     ->getPublishedNodes<std::string, GeoFullPhysVol *>(q);
 
-    auto matches = [&](const std::string &name, const GeoVFullPhysVol &fpv) {
-      bool match = m_cfg.nameList.empty() && m_cfg.materialList.empty();
-      if (match) {
+    auto matches = [&](const std::string &name, PVConstLink physvol) {
+      if (m_cfg.nameList.empty() && m_cfg.materialList.empty()) {
         return true;
       }
 
-      match |= std::any_of(
+      bool matchName{false}, matchMaterial{false};
```

Comment:
```suggestion
      bool matchName{false};
      bool matchMaterial{false};
```

nitpick; clang-tidy might also pick on that

Reply:
actually I removed those:)